django_ticketoffice/api.py

```diff
@@ -1,9 +1,27 @@
 # -*- coding: utf-8 -*-
 """API shortcuts for use in client applications."""
-from django_ticketoffice.exceptions import NoTicketError  # NoQA
-from django_ticketoffice.exceptions import CredentialsError  # NoQA
-from django_ticketoffice.exceptions import TicketExpiredError  # NoQA
-from django_ticketoffice.exceptions import TicketUsedError  # NoQA
-from django_ticketoffice.models import Ticket  # NoQA
-from django_ticketoffice.decorators import invitation_required  # NoQA
-from django_ticketoffice.decorators import stamp_invitation  # NoQA
+from django_ticketoffice.exceptions import (
+    CredentialsError,
+    NoTicketError,
+    TicketExpiredError,
+    TicketUsedError,
+)
+from django_ticketoffice.decorators import (
+    invitation_required,
+    stamp_invitation,
+)
+from django_ticketoffice.models import Ticket
+
+
+__all__ = [
+    # exceptions
+    CredentialsError,
+    NoTicketError,
+    TicketExpiredError,
+    TicketUsedError,
+    # decorators
+    invitation_required,
+    stamp_invitation,
+    # models
+    Ticket,
+]
```

Comment:
__all__ expects strings, not classes. It's a bit weird, but I've seen errors when doing that.